src/components/DocumentControls/DocumentControls.js

```diff
@@ -91,15 +124,7 @@ const DocumentControls = props => {
 
   const extractPages = () => {
     if (selectedPageIndexes.length === 0) {
-      const warning = {
-        message: t('option.thumbnailPanel.extractZeroPageError'),
```

Comment:
Should remove `extractZeroPageError` from json translation files. Beside that I think this PR look good

Reply:
Resolved in 991edca